providers/cncf/kubernetes/src/airflow/providers/cncf/kubernetes/triggers/job.py

```diff
@@ -69,7 +73,18 @@ def __init__(
         super().__init__()
         self.job_name = job_name
         self.job_namespace = job_namespace
-        self.pod_name = pod_name
+        if pod_name is not None:
+            warnings.warn(
+                "`pod_name` parameter is deprecated, please use `pod_names`",
+                AirflowProviderDeprecationWarning,
+                stacklevel=2,
+            )
+            self.pod_name = pod_name
```

Comment:
```suggestion
```
Instead of initializing it here, make `self.pod_name` a read-only property (see previous suggestion for `self.pod` of the operator).

Reply:
Done! Thank you for your review!